beast/tests/helpers.py

```diff
@@ -46,11 +48,19 @@ def compare_tables(table_cache, table_new):
         # test numerical types for closeness
         # and other types for equality
         if table_new[tcolname].data.dtype.kind in ["f", "i"]:
-            np.testing.assert_allclose(
-                table_new[tcolname],
-                table_cache[tcolname],
-                err_msg=("%s columns not equal" % tcolname),
-            )
+            if rtol is None:
+                np.testing.assert_allclose(
+                    table_new[tcolname],
+                    table_cache[tcolname],
+                    err_msg=("%s columns not equal" % tcolname),
+                )
+            else:
+                np.testing.assert_allclose(
+                    table_new[tcolname],
+                    table_cache[tcolname],
+                    rtol=rtol,
+                    err_msg=("%s columns not equal" % tcolname),
+                )
```

Comment:
The if statement is not needed.  You can just use the statement in the `else` portion as if `rtol=None` then it is like not setting rtol at all.

Reply:
I tried doing exactly that, but when `rtol=None`, `np.testing.assert_allclose` throws an error saying a float is required.

```
>>> np.testing.assert_allclose(1, 1.5, rtol=None)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-11-21697138e8e1> in <module>()
----> 1 np.testing.assert_allclose(1, 1.5, rtol=None)

~/miniconda3/envs/ac/lib/python3.5/site-packages/numpy/testing/_private/utils.py in assert_allclose(actual, desired, rtol, atol, equal_nan, err_msg, verbose)
   1489
   1490     actual, desired = np.asanyarray(actual), np.asanyarray(desired)
-> 1491     header = 'Not equal to tolerance rtol=%g, atol=%g' % (rtol, atol)
   1492     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
   1493                          verbose=verbose, header=header, equal_nan=equal_nan)

TypeError: a float is required
```

Reply:
That is not what I expected.  Weird.  Then I would suggest setting the default to 1e-7 as this is the same as the default rtol for this function.  Better than having the code duplicated I would say.

Reply:
sounds good!